counsel-osx-app.el

```diff
@@ -39,7 +39,7 @@
 (require 'ivy)
 
 (defvar counsel-osx-app-location "/Applications"
-  "Path to directory containing applications.")
+  "Path (or list of paths) to directories containing applications.")
```

Comment:
That's funny because I never used as a string, but as list of strings. Thanks for fixing documentation 💃